install_config/redeploying_certificates.adoc

```diff
@@ -773,10 +773,10 @@ $ oc adm certificate approve <csr_name> <1>
 ----
 <1> `<csr_name>` is the name of a CSR from the list of current CSRs.
 
-* Approve all CSRs:
+* Approve only Pending CSRs:
```

Comment:
Suggest changing "only Pending" to "all pending"

Reply:
ok, done